docs/modules/ROOT/pages/create.adoc

```diff
@@ -0,0 +1,71 @@
+== How to Create a Starter Kit
+
+=== What is a Starter Kit?
+
+https://openzeppelin.com/starter-kits/[Starter Kit] is a preconfigured Ethereum project containing the best industry tools to bootstrap a dApp development process.
+
+=== Why Should I Create a Starter Kit?
+
+Anyone can create a Starter Kit and it is a great way to introduce developers to a new tool or a project, like https://github.com/OpenZeppelin/starter-kit-gsn[GSN Starter Kit].
```

Comment:
I'd expand a bit more here, especially on the ethos behind kits, in that the idea is to provide devs with a box with everything ready to hit the road running when building a project.